settings.py

```diff
@@ -1480,7 +1480,6 @@ def _pkce_required(client_id):
     INSTALLED_APPS = INSTALLED_APPS + ('corehq.apps.mocha',)
     import warnings
     warnings.simplefilter('default')
-    os.environ['PYTHONWARNINGS'] = 'd'  # Show DeprecationWarning
```

Comment:
As I was reviewing the python `warning` docs to brush up for reviewing this PR, I came across https://docs.python.org/3/library/warnings.html#overriding-the-default-filter (see the second example). There, this line comes with the comment `# Also affect subprocesses`, which implies to me that the original purpose of this line was to make this change (if changing to default is a change at all) affect subprocesses as well.

Just wanted to raise that because I remember we had a conversation where neither of us was clear on what this point of setting the environment variable here was.